README.md

```diff
@@ -1 +1,147 @@
+- [📺 Presentation](#presentation)
+- [📝 License](#license)
+
+# 💻 wsc2022 <a name="about-project"></a>
+
+This is my capstone project for module 1 of Microverse program. In this project, I built a website for a world snooker championship, where people can see about the event, top players and other programs.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+    <li>[Bootstrap 5](https://getbootstrap.com/)</li>
+    <li>[Font Awesome](https://fontawesome.com/)</li>
+    <li>[Lato Fonts](https://www.latofonts.com/)</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Mobile-and-Desktop Website**
+- **Responsive Layout**
+- **Dynamic Contents**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+You can see my project [live demo](https://app-developer8866.github.io/wsc2022.github.io/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, clone this repository to your desired folder using the following command:
+
+```sh
+  git clone git@github.com:App-Developer8866/Capstone1.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👨‍🚀 IndieCoderMM <a name="authors"></a>
+
+- GitHub: [@App-Developer8866](https://github.com/App-Developer8866/)
+- LinkedIn: [Muhammad Muttraf Rasool](https://www.linkedin.com/in/muhammad-muttraf-rasool-421819202)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add dynamic button for mobile**
+- [ ] **Add form for ordering tickets**
+- [ ] **Create ranking table of top players**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, you can support me by giving a ⭐.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+The design inspiration of this project is from [CC Global Summit 2015](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015/modules/192118207) created by [Cindy Shin in Behance](https://www.behance.net/adagio07).
+
+I would like to thank all of the code reviewers for making my project better.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📺 Presentation <a name="presentation"></a>
+
+You can watch my [presentation about this project here](https://www.loom.com/share/ea2dbf75e5964a2c8bdccec9b1c98b58).
+
```

Comment:
- [Optional] Dear @App-Developer8866, Good job making the video presentation, It would be nice to have it on the pull request description so that your reviewers would find it easily.